source/components/cardContainer/filters/filterGroup/filterGroup.directive.ng1.ts

```diff
@@ -54,5 +68,6 @@ export let filterGroup: angular.IComponentOptions = {
 		icon: '=',
 		filterGroup: '=',
 		source: '=',
+		enabled: '<',
```

Comment:
So the case we're generally expecting is `disabled == true`.  If we frequently checked the conditions you stated, then the solution you stated totally makes way more sense!  But...we're hopefully not doing that, since the disabled state should be the exceptional state, not the default state.  (Sam said it well, I just like typing right now)